lib/libfrr.h

```diff
@@ -128,7 +128,8 @@ extern void frr_preinit(struct frr_daemon_info *daemon, int argc, char **argv);
 extern void frr_opt_add(const char *optstr, const struct option *longopts,
 			const char *helpstr);
 extern int frr_getopt(int argc, char *const argv[], int *longindex);
-extern void frr_help_exit(int status);
+
+__attribute__((__noreturn__)) extern void frr_help_exit(int status);
```

Comment:
I bet for moving `__attribute__` in-between extern and void. Just to properly distinguish static/extern stuff. Other than that, I'm fine with this.